nx-dev/ui-home/src/lib/work-better-achieve-more-ship-quicker.tsx

```diff
@@ -0,0 +1,130 @@
+import {
+  ButtonLink,
+  SectionHeading,
+  Strong,
+  TextLink,
+} from '@nx/nx-dev/ui-common';
+import Image from 'next/image';
+
+export function WorkBetterAchieveMoreShipQuicker(): JSX.Element {
+  return (
+    <article className="mx-auto max-w-7xl px-6 lg:px-8">
+      <div className="max-w-5xl">
+        <SectionHeading
+          as="h2"
+          variant="title"
+          id="achieve-more-ship-quicker"
+          className="scroll-mt-24"
+        >
+          Work better together. Achieve more. Ship quicker.
+        </SectionHeading>
+        <SectionHeading as="p" variant="subtitle" className="mt-6">
+          <Strong>Tangled codebases</Strong> without well-defined ownership{' '}
+          <Strong>crush the velocity</Strong> of teams and the quality of
+          products. Nx fixes it. By defining{' '}
```

Comment:
"Nx fixes it" could be replaced with "Nx solves this." Nx provides the solution, and implementing the solution "fixes" the problem.